utils/ripgrep/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ripgrep
+PKG_VERSION:=13.0.0
+PKG_RELEASE:=1
+
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/BurntSushi/ripgrep/archive/refs/tags/
+PKG_HASH:=0fb17aaf285b3eee8ddab17b833af1e190d73de317ff9648751ab0660d763ed2
+
+PKG_BUILD_DEPENDS:=rust/host
+
+include ../../lang/rust/rust-package.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Build/Compile
+	$(call Build/Compile/Cargo,, --features 'pcre2')
+endef
+
+define Package/$(PKG_NAME)
+    SECTION:=utils
+    CATEGORY:=Utilities
+    DEPENDS:=$(RUST_ARCH_DEPENDS) @!SMALL_FLASH @!LOW_MEMORY_FOOTPRINT +libpcre2
```

Comment:
Why do you depend on `SMALL_FLASH` and `LOW_MEMORY_FOOTPRINT`?

Reply:
it is inherited I can remove it.